packages/operator/pkg/apiserver/routes/v1/training/model_training_validation.go

```diff
@@ -195,6 +207,30 @@ func (mtv *MtValidator) validateVCS(mt *training.ModelTraining) (err error) {
 	return
 }
 
+func (mtv *MtValidator) validateObjectStorage(mt *training.ModelTraining) (err error) {
+	objStorage, odahuErr := mtv.connRepository.GetConnection(mt.Spec.AlgorithmSource.ObjectStorage.Connection)
+	if odahuErr != nil {
+		logMT.Error(err, MtObjectStorageNotExistsErrorMessage)
+
+		err = multierr.Append(err, odahuErr)
+		return
```

Comment:
In this case it is always one error
```suggestion
		return odahuErr
```

Same thing for line 220. No need to use `multierr` for one error.